src/amber/controller/helpers/tags.cr

```diff
@@ -0,0 +1,7 @@
+module Amber::Controller::Helpers
```

Comment:
I think the helper::tags should be in extensions or support since its adding functionality